docs/_getting-started/dev-mode.md

```diff
@@ -41,8 +48,8 @@ And check the balance of the account:
 > eth.getBalance(eth.accounts[1])
 ```
 
-If you want to test your dapps with a realistic block time use the `--dev.period` option when you start dev mode:
+If you want to test your dapps with a realistic block time use the `--dev.period` option when you start dev mode with the `--dev.period 14` argument.
 
-```shell
-geth --dev --dev.period 14 console
-```
+## Connect Remix to Geth
+
+With geth now running, open <http://remix.ethereum.org> (Yes, use `http` when connecting to your local machine). Compile the contract as normal, but when you deploy and run a contract, select _Web3 Provider_ from the _Environment_ dropdown, and add "http://127.0.0.1:8545" to the popup box. Click _Deploy_, and interact with the contract. You should see contract creation, mining, and transaction activity.
```

Comment:
I'm not so happy to recommend 'http' over 'https' here. We need to make it work with the https version of remix.

Reply:
@fjl OK, so for a getting started guide, would a simple solution be using something like ngrok or localtunnel? Or do you know a better way to setup secure communication between a website and a local machine, within the scope of a getting started guide?

Reply:
https remix does work with Chrome and local geth if you use `http://127.0.0.1:8584` as the provider URL. It does not work if you use `localhost` in the provider URL, which is still the default in remix AFAIK.

Reply:
I wish we could find a way to make it work in Firefox as well.

Reply:
Can't we just merge this @fjl